ext/gettext/config.m4

```diff
@@ -23,6 +26,15 @@ if test "$PHP_GETTEXT" != "no"; then
     [AC_CHECK_LIB([c], [bindtextdomain], [
       GETTEXT_LIBS=
       GETTEXT_CHECK_IN_LIB=c
+
+      dnl If libintl.h is provided by libc, it's possible that libc is musl.
+      dnl The gettext family of functions under musl ignores the codeset
+      dnl suffix on directories like "en_US.UTF-8"; instead they look only
+      dnl in "en_US". To accomodate that, we copy some test cases from one
+      dnl to the other.
+      AC_MSG_NOTICE([copying en_US.UTF-8 messages to en_US in case you are on musl])
+      mkdir -p "${srcdir%/}"/ext/gettext/tests/locale/en_US
+      cp -r "${srcdir%/}"/ext/gettext/tests/locale/en_US.UTF-8/* "${srcdir%/}"/ext/gettext/tests/locale/en_US/
```

Comment:
One possible issue with this, is that new files in `en_US.UTF-8/` will not be reflected to `en_US/` until `./configure` is executed again. This may be confusing during test development.

An alternative would be to move this to a Makefile rule, but I'm not sure that we can efficiently trigger the copy only if the contents of `en_US.UTF-8/` changed.

An other one would be to symlink `en_US/` to `en_US.UTF-8/`. I don't know how well symlinks are supported on various systems, but since the `en_US/` path will be used only on Musl this might be fine.

Reply:
An other alternative would to be create the symlink in phpt files directly, and only on Musl. This way we avoid committing a symlink as well a potential Windows issues. We can create a `.inc` file in `ext/opcache/tests` that we include in all relevant tests.

Reply:
In the latest force-push, I changed the `cp -r` to an `ln -s`. That should avoid issues with newly-added test files, and is probably safe: the `ln -s` is only performed if your libc provides `gettext()`, and I doubt that will be true on any system where symlinks don't work.